models/user.go

```diff
@@ -265,7 +267,7 @@ func FindUserByConfirmationToken(tx *storage.Connection, token string) (*User, e
 
 // FindUserByEmailAndAudience finds a user with the matching email and audience.
 func FindUserByEmailAndAudience(tx *storage.Connection, instanceID uuid.UUID, email, aud string) (*User, error) {
-	return findUser(tx, "instance_id = ? and email = ? and aud = ?", instanceID, email, aud)
+	return findUser(tx, "instance_id = ? and email COLLATE utf8mb4_unicode_ci  = ? and aud = ?", instanceID, email, aud)
```

Comment:
I guess that specifying database sensitive logic in search is not good idea. It seems that FindUserByEmailAndAudience function will not work on postgres

Reply:
Thanks a ton for the feedback @icecream78. I suppose just a plain LOWER(email) should do (since its db independent) - that and lowering the email parameter. Please bare with me while I attempt to update this PR instead of creating a new one

Reply:
@i-pip yeah, it's a good idea!